pkg/reconciler/reconciler_test.go

```diff
@@ -217,11 +218,11 @@ var _ = Describe("Reconciler", func() {
 		var _ = Describe("WithMaxReleaseHistory", func() {
 			It("should set the max history size", func() {
 				Expect(WithMaxReleaseHistory(10)(r)).To(Succeed())
-				Expect(r.maxHistory).To(Equal(10))
+				Expect(r.maxReleaseHistory).To(PointTo(Equal(10)))
 			})
 			It("should allow setting the history to unlimited", func() {
 				Expect(WithMaxReleaseHistory(0)(r)).To(Succeed())
-				Expect(r.maxHistory).To(Equal(0))
+				Expect(r.maxReleaseHistory).To(PointTo(Equal(0)))
```

Comment:
Is there some place that we can test that the default value is 10 when `WithMaxReleaseHistory()` is not used.

Reply:
I looked briefly and didnt see anywhere, ill take a closer look and write a new spec if i have to